codemods/jscodeshift/next/13/new-link/README.md

```diff
@@ -0,0 +1,21 @@
+Safely removes `<a>` from `next/link` or adds `legacyBehavior` prop.

```

Comment:
from `Link` components imported from `next/link` or adds the `legacyBehavior` prop on the component level.